server/channels/app/user.go

```diff
@@ -54,7 +54,7 @@ func (a *App) CreateUserWithToken(c request.CTX, user *model.User, token *model.
 
```

Comment:
Hey :wave:  @hanzei . QQ: On L51, shouldn't that be an 'or' operation, since a token can only have one type at any given time, that expression will never evaluate to true, saw that and was curious.

Reply:
Okay, got it. :+1: